brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Call To Action - Add 2nd Button To CTA.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/ed1d9717-0ff7-4106-acec-7c60f1fd03a9?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- SS - https://share.getcloudapp.com/bLuKKvW4

### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Create a CTA in master v1.25.4 and switch to our branch. ( No validation issue after editor refresh )
- Drag and drop the CTA block.
- Go to Buttons > Enable 2nd CTA option. You will get tab 2nd Button.
- Try out the option provided. Check whether it is working fine or not ( Editor as well as front ).
- Try to create a demo using CTA mentioned in Ss.

### Checklist:
